app/redux/reducers/SettingsReducer.js

```diff
@@ -16,13 +16,15 @@ const defaultState = {
     endSetTimerDuration: 30,
     isEditingEndSetTimer: false,
     syncDate: '',
+    wasEdited: false
```

Comment:
need a wasEdited for both default metric and defult timer

Reply:
Done